solarforecastarbiter/datamodel.py

```diff
@@ -1255,10 +1257,9 @@ def from_dict(model, input_dict, raise_on_extra=False):
         dict_ = input_dict.copy()
         if model != ReportFigure:
             return super().from_dict(dict_, raise_on_extra)
-        figure_class = dict_.get('figure_class')
-        if figure_class == 'plotly':
```

Comment:
eee, maybe making init false was a bad idea. I prefer using the figure class to determine this, so probably best to remove that, but keep the parameter at the end